packages/teams-js/src/public/secondaryBrowser.ts

```diff
@@ -6,14 +6,20 @@ import { ErrorCode } from './interfaces';
 import { runtime } from './runtime';
 
 /**
- * Namespace to power up the in-app browser experiences in the Host App.
- * For e.g., opening a URL in the Host App inside a browser
+ * Namespace to power up the in-app browser experiences in the host app.
+ * For e.g., opening a URL in the host app inside a browser
+ * Namespace to power up the in-app browser experiences in the host app.
+ * For e.g., opening a URL in the host app inside a browser
  *
  * @beta
  */
 export namespace secondaryBrowser {
   /**
-   * Open a URL in the secondary browser aka in-app browser
+   * Open a URL in the secondary browser.
+   *  
+   * On mobile, this is the in-app browser.
+   * 
+   * On web and desktop, the user will be redirected to an external browser.
```

Comment:
It's awesome that you're helping with documentation but unfortunately this is not correct, there is currently not going to be support for web and desktop as there is no user need that this is fulfilling

Reply:
@alicialu-MSFT is the web and desktop behavior (redirection to external browser) the current fallback behavior, or is open() completely no-op for web/desktop?

Reply:
Yup, it's a no-op. Could we change the language to say "On web and desktop, please use the Window.open() method or other native external browser methods."